cinder/volume/drivers/nexenta/jsonrpc.py

```diff
@@ -28,6 +28,13 @@
 
 LOG = logging.getLogger(__name__)
 
+REST_PORT = 8457
```

Comment:
maybe change to DEFAULT_REST_PORT?

Reply:
Changed to DEFAULT_REST_PORT, thank you!